src/shipit_static_analysis/tests/test_clang.py

```diff
@@ -159,7 +155,7 @@ def test_clang_tidy(mock_repository, mock_config, mock_clang, mock_stats, mock_r
 
     metrics = mock_stats.get_metrics('issues.clang-tidy.publishable')
     assert len(metrics) == 1
-    assert metrics[0][1] == 0
+    assert metrics[0][1] == 2
```

Comment:
This test result was bad *before*, now it's giving the valid result (a different config was used when triggering `is_publishable`...)